session.go

```diff
@@ -265,7 +265,6 @@ runLoop:
 					continue
 				}
 				s.close(err)
-				break runLoop
```

Comment:
I'm not sure this is the right fix – I'd be more happy with a `continue`. That way, we don't enter the code at the bottom of the run loop (e.g. sending packets). Keep in mind that this error here may be triggered by a peer doing something security-relevant, so I don't think we should do much more work other than sending the close frame.